common/religion/religions/titanic_group.txt

```diff
@@ -47,8 +47,8 @@
 
 	# Placeholder
 	traits = {
-		virtues = { generous compassionate calm }
-		sins = { greedy sadistic callous wrathful }
+		virtues = { just diligent callous }
+		sins = { arbitrary lazy compassionate }
```

Comment:
I still don't think they should have callous as a virtue and compassionate as a sin.
They're pragmatic, not assholes.

Wrathful sin and calm virtue is more fitting.